BAZProjectC1/Components/BasicInfoMovie/movieImageView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  movieImageView.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 28/09/22.
+//
+
+import Foundation
+import UIKit
+
+open class movieImageView: UIView {
+    
+    public lazy var posterImage: UIImageView = {
```

Comment:
Es necesario que esta propiedad sea pública?
No encontré algún lugar donde se esté utilizando. Recuerda que al usar public estás comunicando la intención de que esta propiedad pueda ser utilizada no solo fuera de tu clase sino también fuera de tu módulo. Además de permitir ser leída e incluso mutada. 

Ten cuidado con esos detalles.

```suggestion
    private lazy var posterImage: UIImageView = {
```